android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -349,6 +349,22 @@ class BrowserToolbar @JvmOverloads constructor(
         display.showMenuButton()
     }
 
+    /**
+     * Hides the page action separator in display/edit mode.
+     */
+    override fun hidePageActionSeparator() {
```

Comment:
Could you add tests for this in BrowserToolbarTest

Reply:
Done.